kfdefs/base/monitoring/kustomization.yaml

```diff
@@ -10,3 +10,4 @@ resources:
   - alertmanager-config.yaml
   - ./blackbox-exporter
   - ./thanos-query
+  - ../../../../grafana/overlays/moc/smaug/blackbox-exporter/service-monitor.yaml
```

Comment:
This has a problem I suspect

Reply:
you need to remove it from the kfdefs directory, the blackbox exporter will now go in the [grafana moc/smaug kustomization](https://github.com/operate-first/apps/blob/master/grafana/overlays/moc/smaug/kustomization.yaml)